dstep/Configuration.d

```diff
@@ -77,6 +77,10 @@ struct Configuration
     /// do not translate following symbols
     @("skip", "Skip translation of <symbol>.")
     string[] skipSymbols;
+
+    /// enable printing of warnings
+    @("enable-warnings", "Enable printing of warnings [default].")
+    bool enableWarnings = true;
```

Comment:
I was thinking about something similar, but I do not like the term
'diagnostic issue', it doesn't express the spirit of the problem exactly.
But I've just had an idea. What about ´ambiguity-action´ and "Action to
take when an ambiguity is encountered.". My point it is not the diagnostics
that needs an action, it is some kind of conflict/ambiguity.

02.09.2016 15:44 "jacob-carlborg" <notifications@github.com> napisał(a):

> In dstep/Configuration.d
> <https://github.com/jacob-carlborg/dstep/pull/95#discussion_r77347957>:
>
> > @@ -77,6 +77,10 @@ struct Configuration
> >      /// do not translate following symbols
> >      @("skip", "Skip translation of <symbol>.")
> >      string[] skipSymbols;
> > +
> > +    /// enable printing of warnings
> > +    @("enable-warnings", "Enable printing of warnings [default].")
> > +    bool enableWarnings = true;
>
> "Action to take when a diagnostic issue is encountered", perhaps?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jacob-carlborg/dstep/pull/95/files/f7cc5b685291575bc8bae2c72ce578b2f85431b9#r77347957>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHfoNOy9YHDg6r0pd7cRFjwV4EUGnizqks5qmChEgaJpZM4JyGgj>
> .
>